db/cache/charts.go

```diff
@@ -538,13 +558,27 @@ func (charts *ChartData) Lengthen() error {
 			days.NewAtoms = append(days.NewAtoms, blocks.NewAtoms.Sum(interval[0], interval[1]))
 			days.Chainwork = append(days.Chainwork, blocks.Chainwork[interval[1]])
 			days.Fees = append(days.Fees, blocks.Fees.Sum(interval[0], interval[1]))
+			days.TotalMixed = append(days.TotalMixed, blocks.TotalMixed.Sum(interval[0], interval[1]))
+
+			var anonymitySet int64
+			for iu := range charts.MixedVouts.Values {
+				maxHeight := blocks.Height[interval[1]]
+
+				if uint64(charts.MixedVouts.FundingHeight[iu]) < maxHeight &&
+					(uint64(charts.MixedVouts.SpendingHeights[iu]) > maxHeight || charts.MixedVouts.SpendingHeights[iu] == -1) {
+
+					anonymitySet += charts.MixedVouts.Values[iu]
+				}
+			}
```

Comment:
I don't understand what this bit is supposed to be doing.  Unlike the other day data sets, which are a sum or average over an interval, this just seems to be getting the anon set size at the end of the interval.  But this data is already computed and stored in the `blocks.AnonymitySet` slice.

If you really wanted to compute the average in this interval, it should be done similar to the other data sets.

Reply:
I am now taking daily average. Thanks for pointing to the fact that the data already stored in `blocks.AnonymitySet`